docs/onduty/onduty.md

```diff
@@ -0,0 +1,54 @@
+# Community On-Duty
```

Comment:
i appreciate us not using "on-call" as the terminology here, since that would imply immediate and off hour reactive actions